bus/broadcast_consumer_connection.go

```diff
@@ -52,6 +53,26 @@ func (b *Bus) doUpdateConnection(_ context.Context, msg *Message) error {
 }
 
 func (b *Bus) doDeleteConnection(ctx context.Context, msg *Message) error {
+	// Ensure this connection isn't being used by any dynamic replays
```

Comment:
We should update the gRPC delete handler to have the same logic. I like this approach a lot better.

Reply:
Totally missed that, done!